### ISSUE TITLE
[Bug]: unsaved post content prompt bypassed

### ISSUE THREAD

**Issue by atomGit** [GetPublii/Publii#1246]
### Operating system

manjaro

### Publii version

0.42.1-1 (AUR)

### Post editor

WYSIWYG editor

### Bug description

if you create a new post and use a button combo (mouse aux for ex.) to go back to the main posts list, the warning prompt regarding unsaved changes is bypassed and the post content is lost

### Steps to reproduce

as above

**Comment by dziudek** [comment]
Hi,

It seems that you somehow triggers a navigation which skips onbeforeunload event which is responsible for the data save popup. 

I am unable to reproduce this case. Also I suppose that there is no way to protect before such behavior or it is just a bug in electron version for linux.

**Comment by qc00** [comment]
This bug exists on Mac OS as well. Simple mouse back button will trigger this, without any prompt.

I just lost an hour of work due to this.

**Comment by atomGit** [comment]
yeah, i remember losing my work when i first encountered this and it's still present in 0.46.5-1 (Arch/Manjaro AUR)

from the build file:
```
# Maintainer: Julian Xhokaxhiu <info at julianxhokaxhiu dot com>
pkgname=publii
pkgver=0.46.5
pkgrel=1
pkgdesc="Publii is a desktop-based CMS for Windows, Mac and Linux that makes creating static websites fast and hassle-free, even for beginners"
arch=('x86_64')
url="https://github.com/GetPublii/Publii"
license=('MIT')
source=("https://getpublii.com/download/Publii-$pkgver.rpm")
sha512sums=('32260e650734b90943459d22cd69fb7839593bdf7ddfa83c35577001c5364be7ee97005d98db63469d4b56d2cf681b2164e9a65b2d210104e2cd525c226c76b4')

package() {
  bsdtar -xpf "${srcdir}/Publii-$pkgver.rpm" -C $pkgdir
  [ -d $pkgdir/usr/lib/.build-id ] && rm -rf $pkgdir/usr/lib/.build-id
}
```

**Comment by dziudek** [comment]
I have added a fix which should block back/forward buttons on mouses in v.0.47.2

**Comment by bobmitro** [comment]
The new Publii v.0.47.2 has been released ([changelog](https://getpublii.com/blog/release-047.html#hotfix-0-47-2)).

**Comment by GamertechAU** [comment]
0.47.5, this bug still exists @dziudek on Linux rpm.
